toolflow/vivado/platform/vc709/vc709.benchmark

```diff
@@ -1,125 +0,0 @@
-{
```

Comment:
This file was deleted, because for some reason it is not consistent with the internal representation of benchmark-Results. Thus, the json-parser will throw multiple errors, due to missing Keys.
I can either fix the file by hand or we could regenerate the results in the future.

Reply:
Ok, now I understand the relation to the JSON-stuff.